### ISSUE TITLE
{Freeway-Bug}[Adaptive Cards - Blog]: Elements marked as presentational should not have global ARIA or tabindex to ensure all screen readers ignore them.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8542]
### Target Platforms

Other

### SDK Version

1.5

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
OS: Version 23H2 (OS Build 25381.1)
URL: https://adaptivecards.io/blog/
Browser: Edge Dev Version 115.0.1880.3 (Official build) dev (64-bit)
Tool: Accessibility Insight For Web

**Repro Steps:**

1. Open the above URL in the edge dev browser.
2. Navigate to 'Blog' tab item and press enter key to activate it.
3. Run the AI Tool and observe the issue.

**Actual Result:**
Elements marked as presentational  have global ARIA or tabindex.

**Expected Result:**
Elements marked as presentational should not have global ARIA or tabindex to ensure all screen readers ignore them

**Element path:**
 .w3-cell-row.w3-padding-16:nth-child(1) > article:nth-child(1) > .w3-margin-top > .w3-cell-middle.w3-cell > a[href$="adaptivecards.io"][role="presentation"]

**How to fix:** 
Fix all of the following:
  Element is not focusable.

**Note:**
This issue is repro throughout the Blog screen.

**User Impact:**
It will impact the AT user if the element marked as presentational have Global Aria or Tab Index.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships

“Have feedback to share on Bugs? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-vaibgu%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400980259288%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=TrcBaCyAPKg0WAQfdSZJP2i9pghMEDg59i2NeUUs5EQ%3D&reserved=0)“

### Screenshots

![WCAG1 3 1-Elements marked as presentational should not have global ARIA or tabindex to ensure all screen readers ignore them](https://github.com/microsoft/AdaptiveCards/assets/95913748/2ef40575-bf86-44e0-82b8-84561e6bcb18)

![image](https://github.com/microsoft/AdaptiveCards/assets/95913748/4196f49a-394e-49f7-9fc4-7d4921d8816d)

### Card JSON

```JSON
NA
```

**Comment by vagpt** [comment]
Hi @anna-dingler,

Could you please let us know by when the changes will be reflected on the prod so that we can reverify this issue at our end as of now this issue is still repro at our end.

Snippet:
![image](https://github.com/microsoft/AdaptiveCards/assets/95913748/6ebe3eb8-9dbe-4c19-bdc4-4d17dd1a759c)

**Comment by anna-dingler** [comment]
Hi @vagpt, we are planning to make a release at the end of the month.

**Comment by vagpt** [comment]
Hi @anna-dingler,

Thanks for the confirmation.

Due to this issue (#8542) and #8541, Adaptive Cards product grade is getting impacted currently it is in Grade D so it would be great if you can release before 26/10/23 so that we can regress this issue and close both issues.

**Comment by anna-dingler** [comment]
Hi @vagpt , a new version of the website was just released!

**Comment by vagpt** [comment]
Thanks @anna-dingler,

Now this issue is working fine on the environment below, hence adding A11yTTValidated tag.

**Test Environment:**
OS: Version 23H2 (OS Build 25967.1010)
URL: https://adaptivecards.io/blog/
Browser: Edge Dev Version 120.0.2172.1 (Official build) dev (64-bit)
Tool: Accessibility Insight For Web

**Snippet:**

![image](https://github.com/microsoft/AdaptiveCards/assets/95913748/7c001dd2-4c4c-4798-b02b-bca9594e5e92)
